src/app/modules/news/pages/news/news.component.scss

```diff
@@ -58,29 +61,58 @@
                 grid-template-columns: 1fr 30rem;
 
                 app-sidenav {
-                    border-left: 1px solid gray;
+                    border-left: 1px solid lightgray;
                 }
 
-                section {
-                    display: grid;
-                    grid-template-columns: 1fr 1fr;
-                    gap: 3rem;
+            }
+        }
+    }
+}
 
-                    .card {
-                        width: 400px;
+@media (max-width: 1280px) {
+    :host {
+        .content-container {
+            .main-container {
+                .news-container {
+                    grid-template-columns: 1fr;
+                    grid-template-rows: 1fr auto;
+                    
```

Comment:
You can set the bottom border radius to 0 so that the purple gap does not appear.